app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+  def show
+    @hotel = Hotel.find(params[:id])
+    @guests = @hotel.guest_names
+  end
+end
```

Comment:
I know you were currently working through the extension before running out of time so I won't dwell on this much but you have `::guest_names` defined as a class method in the `Hotel` model, but here you are using it as an instance method.

Without knowing how far you got, I think you want this to be an instance method.